internal/gitstream/assign.go

```diff
@@ -49,28 +51,24 @@ func (a *Assign) Run(ctx context.Context) error {
 	return nil
 }
 
-func (a *Assign) getOwnersContent(ctx context.Context) (*Owners, error) {
+func (a *Assign) getOwnersContent() (*Owners, error) {
 
-	content, _, _, err := a.GC.Repositories.GetContents(ctx, a.RepoName.Owner, a.RepoName.Repo, ownersFile, nil)
-	if err != nil || content == nil { // `content` can be nil if the filePath refers to a directory
-		return nil, fmt.Errorf("could not get %s's content from %s/%s: %v", ownersFile, a.RepoName.Owner, a.RepoName.Repo, err)
-	}
-	data, err := content.GetContent()
+	data, err := a.FileSystemHelper.ReadFile(ownersFile)
```

Comment:
I think I would instead use `yaml.NewDecoder()` to avoid storing all the file's bytes in memory (even though it's quite small).

Reply:
Fixed.